tests/unit/project_loader/extensions/test_utils.py

```diff
@@ -195,7 +197,7 @@ def test_multiple_extensions(self):
         self.expectThat(config.data["environment"], Contains("TEST_EXTENSION"))
 
 
-class ExtensionOrderConsistencyTest(ExtensionTestBase):
+class MultipleExtensionsExtendPlugs(ExtensionTestBase):
```

Comment:
we changed the order, but it does not mean we should change the test this way.
Ideally the scenario should include the expected result and it should represent the order in which the extensions were added.

Reply:
Done!